relations/controllers.py

```diff
@@ -147,6 +147,11 @@ def supercede(arxiv_id_str: str,
         raise InternalServerError("The previous relation cannot be found") \
             from nfe
 
+    except DBLookUpError as lue:
+        raise InternalServerError("A failure occured in "
+                                  + "looking up the previous relation") \
```

Comment:
```suggestion
                                 "looking up the previous relation") \
```

Shouldn't need ``+`` within parentheses